src/Client.php

```diff
@@ -34,9 +34,9 @@ class Client
      *
      * @codeCoverageIgnore
      * @param integer $size
-     * @param GeneratorInterface $generator
+     * @param GeneratorInterface|null $generator
      */
-    public function __construct($size = 21, GeneratorInterface $generator = null)
+    public function __construct($size = 21, ?GeneratorInterface $generator = null)
```

Comment:
If keeping backwards compatibility with older PHP versions is a must, this could be addressed by removing the native typehint and relying on the PHPDoc instead.

```suggestion
    public function __construct($size = 21, $generator = null)
```

Reply:
@acelaya Thx for your contribution, LGTM.